src/utils.ts

```diff
@@ -388,6 +388,14 @@ export async function createRedis(serverConfig: PluginsServerConfig): Promise<Re
     return redis
 }
 
+export function pluginDigest(plugin: Plugin): string {
```

Comment:
This seems dead

Reply:
Merge conflict resolution gone wrong, fixed